docs/getting-started.md

```diff
@@ -66,10 +66,10 @@ mv ./clarinet /usr/local/bin
 >**_NOTE:_**
 >
 >If you are using macOS, you may receive security errors when trying to run the pre-compiled binary. 
->You can resolve the security warning by using the command below.
+>In the command mentioned below, update the <path-to-the-downloaded-client-library> with your library path and then use the command to resolve the security warning.
 
 ```sh
-xattr -d com.apple.quarantine /path/to/downloaded/clarinet/binary
+xattr -d com.apple.quarantine <path-to-the-downloaded-client-library>
```

Comment:
"library path" is still confusing. How about:

To resolve this error, use the command below, replacing `<path-to-clarinet-binary>` with the path to the downloaded `clarinet` binary (e.g. `/usr/local/bin/clarinet`).

```sh
xattr -d com.apple.quarantine <path-to-clarinet-binary>
```

Reply:
Done